src/databases/Vs/avtVsFileFormat.h

```diff
@@ -183,6 +183,13 @@ class avtVsFileFormat: public avtSTMDFileFormat {
    */
   HighOrderUnstructuredData thisData;
 
+  /**
```

Comment:
These are doxygen comments, so that one can autogenerate documentation.  Do you have a different way that you like to see doxygen comments?

Also, I reformatted this one file according to our standards, noted in the last comment.  Pls LMK whether that formatting is acceptable.  (If it is not, then we will likely need to fix all the files of this plugin, even in the trunk.)